core/test/utils/assertions.hpp

```diff
@@ -680,14 +680,14 @@ namespace detail {
 
 
 template <typename T>
-std::initializer_list<std::initializer_list<T>> l(
-    std::initializer_list<std::initializer_list<T>> list)
+const std::initializer_list<std::initializer_list<T>> &l(
+    const std::initializer_list<std::initializer_list<T>> &list)
```

Comment:
Could you explain these changes? avoid unnecessary copy?

Reply:
Oh yeah, I forgot to mention these changes, sorry! They are supposed to fix the -Winit-list-lifetime warnings from GCC >= 9, see https://github.com/ginkgo-project/ginkgo/pull/662